src/main/java/org/spongepowered/mod/effect/particle/SpongeParticleHelper.java

```diff
@@ -139,7 +139,8 @@
             float note = ((SpongeParticleEffect.Note) effect).getNote();
 
             if (note == 0f) {
-                return Lists.<Packet>newArrayList(new S2APacketParticles(internal, true, (float) px, (float) py, (float) pz, (float) ox, (float) oy, (float) oz, 0f, count, extra));
+                return Lists.<Packet>newArrayList(new S2APacketParticles(internal, true, px, py, pz, (float) ox, (float) oy, (float) oz, 0f,
+                        count, extra));
```

Comment:
yeah, I kinda went through and did everything but working on top of my previous changes. It's in a separate commit tho, so just look at the more relevant commits.